src/dmd/backend/oper.d

```diff
@@ -649,7 +649,7 @@ immutable ubyte[OPMAX] opcost =
     return tab;
 } ();
 
-extern (C++) __gshared const(char)*[OPMAX] debtab =
```

Comment:
@WalterBright can you confirm if this is being exported for external usage or really not needed? This makes `oper_str` a pure call.

Reply:
It might be used in DMC, but there is a lot of stuff in the backend which has been refactored-to-death (i.e. made redundant) which should be removed.